<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reviewFrame</class>
 <widget class="QWidget" name="reviewFrame">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>review data</string>
  </property>
  <widget class="QPushButton" name="btnSaveFile">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Save
NVM file</string>
   </property>
  </widget>
  <widget class="Line" name="lineHTop">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>800</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelFrameTitle">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>15</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>NVM data
review</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="Line" name="lineVTop">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>3</width>
     <height>80</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReturn">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoadFile">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Load
NVM file</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>781</width>
     <height>501</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="rowCount">
    <number>3</number>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>26</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>26</number>
   </attribute>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>No.</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Signal name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Update or not</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Old value</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>New value</string>
    </property>
   </column>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnReturn</sender>
   <signal>clicked()</signal>
   <receiver>reviewFrame</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>710</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>758</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSaveFile</sender>
   <signal>clicked()</signal>
   <receiver>reviewFrame</receiver>
   <slot>saveFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLoadFile</sender>
   <signal>clicked()</signal>
   <receiver>reviewFrame</receiver>
   <slot>loadFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>564</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadFile()</slot>
  <slot>saveFile()</slot>
 </slots>
</ui>
